src/config.h

```diff
@@ -391,6 +391,13 @@ void setcpuaffinity(const char *cpulist);
 #define ATTRIBUTE_TARGET_AVX512
 #endif
 
+/* Check if we can compile ARM SIMD code */
+#if defined(__ARM_NEON) || defined(__ARM_NEON__)
```

Comment:
I see we also have a check for `__ARM_NEON` and some places we use `#ifdef __aarch64__`. Can we clean these up so they all use neon based off of the same check.